src/export/export-tailwind.ts

```diff
@@ -0,0 +1,172 @@
+import fs from 'fs-extra';
+import path from 'path';
+import prettier from 'prettier';
+
+import { version } from '../../package.json';
+import { renderTemplate } from './templating';
+
+import type {
+  Tokens,
+  TailwindConfig,
+  TextVariant,
+  Breakpoints,
+  Dictionary,
+} from '../utils/types';
+
+const prettierConfigFile = path.resolve(__dirname, '../../.prettierrc');
+const templateDir = path.resolve(__dirname, '../../templates');
+
+// Run Prettier on JS code using the config file
+const formatFileContents = async (contents: string) => {
+  return prettier.resolveConfig(prettierConfigFile).then((options) => {
+    return prettier.format(contents, { ...options, parser: 'babel' });
+  });
+};
+
+export const styleValueAtBreakpoint = (
+  key: string,
+  value: string | string[] | Dictionary<string>
+): string => {
+  if (typeof value === 'string') {
+    return value;
+  }
+  if (Array.isArray(value)) {
+    throw new Error('Cannot handle array value for responsive textStyles');
+  }
+  return value[key];
+};
+
+// This takes the responsive textStyles that get applied in a system like Chakra
+// and translates them into a set of responsive styles at the various breakpoints
+// defined in the theme.
+export const generateResponsiveTextStyles = (
+  styles: Dictionary<TextVariant>,
+  breakpoints: Breakpoints
+) => {
+  breakpoints = {
```

Comment:
This 0px breakpoint is very silly and would like a sensible way to remove it very soon